### PR TITLE
QA test for relationship card

### PR DESCRIPTION
**Pull request by thfong**

### CODE REVIEW

File: spec/screening/relationship_card_spec.rb [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    # Adding an existing child to a screening
+    within '#search-card', text: 'Search' do
+      autocompleter_fill_in 'Search for any person', person1[:fname]
+      find('li', text: "#{person1[:fname]} #{person1[:lname]}").click
+    end
+
+    within '#relationships-card' do
+      # Verify rendering of card
+      expect(page).to have_content("#{person1[:fname]} #{person1[:lname]} is the...")
+      expect(page).to have_content('Son  of Missy R.')
```
Comment (AmanShurafa) [inline spec/screening/relationship_card_spec.rb]: Is there a reason for the 2 spaces here `Son  of`?

File: spec/screening/relationship_card_spec.rb [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+      expect(page).to have_content('Son of Ricky W.')
+      expect(page).to have_content('Brother of Roland W.')
+      expect(page).to have_content('Brother  of Sharon W.')
```
Comment (AmanShurafa) [inline spec/screening/relationship_card_spec.rb]: 2 spaces here too?

File: spec/screening/relationship_card_spec.rb (hunk 1 above)
Comment (thfong) [inline spec/screening/relationship_card_spec.rb]: Yes, the value from elastic search contains an extra space depending on the record being pulled

File: spec/screening/relationship_card_spec.rb (hunk 2 above)
Comment (thfong) [inline spec/screening/relationship_card_spec.rb]: See previous comment.

**Pull request merged**